library/timing.c

```diff
@@ -304,6 +304,27 @@ unsigned long mbedtls_timing_get_timer( struct mbedtls_timing_hr_time *val, int
     return( delta );
 }
 
+unsigned long long mbedtls_timing_get_timer_us( struct mbedtls_timing_hr_time *val, int reset )
```

Comment:
a new function was introduced, but I don't see the implementation for windows, https://github.com/ARMmbed/mbedtls/blob/development/library/timing.c#L244 for example. This PR cannot be merged without windows implementation, otherwise the library will not compile.